doc/index.rst

```diff
@@ -56,8 +56,6 @@ CCArchiver
 .. image:: https://img.shields.io/badge/ts_xml-dm_CCArchiver-green.svg
     :target: https://ts-xml.lsst.io/sal_interfaces/CCArchiver.html
 
-.. TODO: Delete the note when the page becomes populated
-
 .. Warning::
 
    **This CSC documentation is under development and not ready for active use.**
```

Comment:
Is this true? I think the warning can be removed!

Reply:
Warning removed